test/artifact/inspec_env_test.rb

```diff
@@ -0,0 +1,14 @@
+require_relative "artifact_helper"
+require "open3"
+
+class TestInspecEnv < Minitest::Test
+  parallelize_me!
+  def test_env
+    command = "/bin/inspec env #{TEST_CLI_OPTS}"
+    stdout, stderr, status = Open3.capture3(command)
+
+    assert_empty stderr.sub(/#< CLIXML\n/, "")
+    assert stdout
+    assert status
```

Comment:
At this point, it seems like there's enough repetition that this should be refactored into its own test type and a compound assertion to clean these up.